tutorials/joint_1d_3d_analysis.ipynb

```diff
@@ -0,0 +1,1314 @@
+      "--------- --------- ----- -------------- ---------- --------- ------\n",
+      "    lon_0 8.363e+01   nan            deg        nan       nan   True\n",
+      "    lat_0 2.202e+01   nan            deg -9.000e+01 9.000e+01   True\n",
+      "amplitude 2.112e-01   nan cm-2 s-1 TeV-1        nan       nan  False\n",
+      "reference 5.055e-05   nan            TeV        nan       nan   True\n",
+      "    alpha 3.531e+00   nan                       nan       nan  False\n",
+      "     beta 5.210e-01   nan                       nan       nan  False\n"
+     ]
+    },
+    {
+     "data": {
+      "application/javascript": [
+       "\n",
+       "            setTimeout(function() {\n",
+       "                var nbb_cell_id = 2;\n",
+       "                var nbb_formatted_code = \"FGL4 = SourceCatalog4FGL()\\ncrab = FGL4[\\\"Crab Nebula\\\"]  #\\ncrab_mod = crab.sky_model()\\ncrab_mod.name = \\\"crab\\\"\\ncrab_pos = crab.position.galactic\\n# freeze position\\ncrab_mod.spatial_model.parameters.freeze_all()\\n# change spectrum to LogParabola (default is PowerLaw for the Crab)\\ncrab_mod.spectral_model = LogParabolaSpectralModel(\\n    amplitude=crab.data[\\\"LP_Flux_Density\\\"],\\n    alpha=crab.data[\\\"LP_Index\\\"],\\n    beta=crab.data[\\\"LP_beta\\\"],\\n    reference=crab.data[\\\"Pivot_Energy\\\"],\\n)\\nprint(crab_mod)\";\n",
+       "                var nbb_cells = Jupyter.notebook.get_cells();\n",
+       "                for (var i = 0; i < nbb_cells.length; ++i) {\n",
+       "                    if (nbb_cells[i].input_prompt_number == nbb_cell_id) {\n",
+       "                        nbb_cells[i].set_text(nbb_formatted_code);\n",
+       "                        break;\n",
+       "                    }\n",
+       "                }\n",
+       "            }, 500);\n",
+       "            "
+      ],
+      "text/plain": [
+       "<IPython.core.display.Javascript object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "FGL4 = SourceCatalog4FGL()\n",
+    "crab = FGL4[\"Crab Nebula\"]  #\n",
+    "crab_mod = crab.sky_model()\n",
+    "crab_mod.name = \"crab\"\n",
+    "crab_pos = crab.position.galactic\n",
+    "# freeze position\n",
+    "crab_mod.spatial_model.parameters.freeze_all()\n",
+    "# change spectrum to LogParabola (default is PowerLaw for the Crab)\n",
+    "crab_mod.spectral_model = LogParabolaSpectralModel(\n",
+    "    amplitude=crab.data[\"LP_Flux_Density\"],\n",
+    "    alpha=crab.data[\"LP_Index\"],\n",
+    "    beta=crab.data[\"LP_beta\"],\n",
+    "    reference=crab.data[\"Pivot_Energy\"],\n",
+    ")\n",
+    "print(crab_mod)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We use the events associated to 3FHL catalog in order to a produce counts cube. We select a 10-by-8 degree region centered on the Crab in Galactic coordinates, and photon energies between 10 GeV and 2 TeV.  "
```

Comment:
I would explain that this is not performing a real data reduction of Fermi data but simply taking reduced products from the 3FHL catalog to create our own dataset.

Also a 10x8 degree map seems large for data above 10 GeV no?

Reply:
yes, reduced to 5x4